validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -1242,8 +1240,19 @@ protected void withAlternateClient(Runnable evaluator) {
     }
   }
 
-  protected void mirrorDeviceConfig() {
-    String receivedConfig = actualize(stringify(receivedUpdates.get(CONFIG_SUBTYPE)));
+  protected void mirrorToOtherConfig() {
+    updateConfig("mirroring config " + useAlternateClient);
+    Config target = (Config) receivedUpdates.get(CONFIG_SUBTYPE);
+    // Since this is updating the mirror, use the opposite of what it would be locally.
+    target.system.testing.endpoint_type = useAlternateClient ? null : "alternate";
+    updateMirrorConfig(actualize(stringify(target)));
+  }
+
+  protected void clearOtherConfig() {
+    updateMirrorConfig("{}");
```

Comment:
Comments please. Should this run in all conditions or are there any useful assertions to add that help document what's going on.